docsets/LiveCode/docset.json

```diff
@@ -12,6 +12,10 @@
         {
             "version": "1.0.0",
             "archive": "versions/1.0.0/LiveCode.tgz"
-            }
+            },
+         {
+             "version": "1.0.1",
```

Comment:
Newer versions need to be at the top of this list, not bottom.

Reply:
Just to check. This is the only place where a new version is specified (apart from its actual presence)?

Reply:
Yep